Monitoring in a Box/mib/data/prometheus/config/alerts.yml

```diff
@@ -97,4 +97,51 @@ groups:
       summary: "Disk usage on / >70%"
       message: "{{ $labels.instance }}: / disk usage >70%. Current usage is {{ humanize $value }}."
 
+# CRITICAL alerts for windows server
+- name: windows_server_critical
+  rules:
+  - alert: windows_server_high_cpu_load_critical
+    expr: 100 - (avg by (instance) (rate(wmi_cpu_time_total{mode="idle"}[2m])) * 100) > 80
```

Comment:
Why is `rate == 2m`? Are you aware of the [difference between `irate` and `rate`](https://utcc.utoronto.ca/~cks/space/blog/sysadmin/PrometheusRateVsIrate)?

Reply:
I have updated on the a file. Understood that irate takes the last two points within its range, which is instant.